doc/source/user_guide/ScienceWindowList.ipynb

```diff
@@ -150,30 +142,10 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
    "id": "81cacabf",
    "metadata": {},
-   "outputs": [
```

Comment:
Do outputs regenerate in CI/CD in these notebooks? Otherwise, we need to keep outputs for the documentation, I guess

Reply:
nope https://oda-api--332.org.readthedocs.build/en/332/user_guide/ScienceWindowList.html

I will add them